kofta/src/app/components/LanguageSelector.tsx

```diff
@@ -67,11 +67,12 @@ export const LanguageSelector: React.FC<LanguageSelectorProps> = ({
     { value: "gsw", label: "🇨🇭 Schwiizerdütsch" }, // Swiss German
 
     /* Other languages */
-    { value: "en-pirate", label: "🏴‍☠️ Pirate" },
+    { value: "en-PIRATE", label: "🏴‍☠️ Pirate" },
```

Comment:
Did your run this locally and check if it fixes the issue?

Reply:
just making sure

Reply:
> Did your run this locally and check if it fixes the issue?

Yep. I made the PR after I was sure that it fixes the issue and doesn't break anything else. All translations are working fine too

Reply:
Sure then it seems fine to me!